source/includes/connection-snippets/x509-snippets/php-connection.php

```diff
@@ -0,0 +1,10 @@
+<?php
+/* Begin x509 connection */
+$certificateKeyFilePath = '/etc/certs/mongodb/client.pem';
+$client = new MongoDB\Client(
+    'mongodb+srv://<cluster-url>/test?authSource=$external&tlsCertificateKeyFile=' . $certificateKeyFilePath . '&retryWrites=true&w=majority&authMechanism=MONGODB-X509'
+);
+$collection = $client->testDB->testCol;
+$docCount = $collection->count([]);
```

Comment:
Same feedback as: https://github.com/mongodb/docs-ecosystem/pull/591#discussion_r352740158

Reply:
Fixed:
* switched to `echo`